Tests/NIOHTTP1Tests/HTTPResponseStatusTests+XCTest.swift

```diff
@@ -27,8 +27,12 @@ extension HTTPResponseStatusTests {
    @available(*, deprecated, message: "not actually deprecated. Just deprecated to allow deprecated tests (which test deprecated functionality) without warnings")
    static var allTests : [(String, (HTTPResponseStatusTests) -> () throws -> Void)] {
       return [
+<<<<<<< Updated upstream
                 ("testHTTPResponseStatusFromStatusCode", testHTTPResponseStatusFromStatusCode),
                 ("testHTTPResponseStatusCodeAndReason", testHTTPResponseStatusCodeAndReason),
+=======
+                ("testHTTPResponseStatusFromStatusCode", testHTTPResponseStatusFromStatusCode), 
+>>>>>>> Stashed changes
```

Comment:
Bad merge again. The best way to fix this one is to run `./scripts/generate_linux_tests.rb`

Reply:
How would I know when I need to run this in the future?

Reply:
Every time you add tests. Also one of the CI jobs ("pull request validation (soundness)") will fail if it needs to be run and hasn't been.